.ai/skills/form.knowledge.md

```
@@ -0,0 +1,209 @@
+| Percent       | `%`        | %       |
+| Respiratory   | `/min`     | breaths/min |
+
+---
+
+## Per-form templates
+
+_Every generated form gets a record here so we can reuse / version it later._
+
+### Template entry format
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)